rockhound/bedmap2.py

```diff
@@ -70,6 +70,14 @@ def fetch_bedmap2(datasets, *, load=True):
         Wether to load the data into an :class:`xarray.Dataset` or just return the
         path to the downloaded data tiff files. If False, will return a list with the
         paths to the files corresponding to *datasets*.
+    chunks : int, tuple or dict
+        Chunk sizes along each dimension. This argument is passed to the
+        :func:`xarray.open_rasterio` function in order to obtain
+        `Dask arrays <https://docs.dask.org/en/latest/array.html>`_ inside the
+        returned :class:`xarray.Dataset`.
+        This helps to read the dataset without loading it entirely into memory.
+    kwargs : dict
```

Comment:
```suggestion
    **kwargs
```

I don't know if this renders correctly in sphinx but it might be better. It's not really passed as a dict when calling the function (you need the `**` if you have a dict).